test/node-rpc-test.js

```diff
@@ -177,6 +177,13 @@ describe('RPC', function() {
       const connectionsCnt = await nclient.execute('getconnectioncount', []);
       assert.strictEqual(connectionsCnt, 1);
     });
+
+    it('should rpc setban a peer',async () => {
```

Comment:
at the beginning of this test, you should get a quick check of current state: assert that there is one peer connected and no peers banned.